src/foam/u2/view/RichChoiceView.js

```diff
@@ -59,6 +63,14 @@ foam.CLASS({
       z-index: 1;
     }
 
+    ^:focus {
+      outline: none;
+    }
+
+    ^:focus ^selection-view {
+      border-color: %SECONDARYCOLOR%;
```

Comment:
consider:
border-color: /*%SECONDARYCOLOR%*/ somereasonablecolor;

So that it has a default value when not used with Nanos.

Reply:
I can do another PR after the colour palette PR merges to do this across the board or just modify that one.

Reply:
Yes, better to do across the board in another PR.

Reply:
Done in https://github.com/foam-framework/foam2/pull/2298.